charts/internal/e2e-resources/README.md

```diff
@@ -35,8 +35,8 @@ This chart creates e2e resources for nri-kubernetes.
 | loadBalancerService.enabled | bool | `true` |  |
 | loadBalancerService.fakeIP | string | `""` |  |
 | pending.enabled | bool | `true` |  |
-| persistentVolume.enabled | bool | `false` |  |
```

Comment:
Could you rebase this branch? I believe the `e2e-rsources` changes are included in main now.

Reply:
Done